### PR TITLE
Added event details page for owned events, logout button, and updated banner

### PR DESCRIPTION
**Pull request by mnugent04**
fixed some css for a couple pages, just made them look more uniform. Added an event details page for events that the user owns, it will allow them to edit events and add roles from there. Also changed the event details page for events the user is invited to has space for accepting an invite. also added logout button on all html pages, and the banner updates based on what page the user is currently viewing.

### CODE REVIEW

File: Tests/SeleniumSetUp.py
Comment (LoganTown) [inline Tests/SeleniumSetUp.py:1]: Why did you remove the chrome manager class?

File: our_site/planner/templates/planner/event_home.html
Comment (LoganTown) [inline our_site/planner/templates/planner/event_home.html:1]: this will 100 percent conflict with me adding a accept or reject button

Comment (mnugent04) [reply our_site/planner/templates/planner/event_home.html:1]: why?

File: Tests/SeleniumSetUp.py
Comment (mnugent04) [reply Tests/SeleniumSetUp.py:1]: oh shoot i forgot i did that i’ll add it back and make a new commit

Comment (mnugent04) [reply Tests/SeleniumSetUp.py:1]: okay its there now

File: our_site/planner/templates/planner/event_modify_invitees.html [hunk 1]
```diff
@@ -1,49 +1,77 @@
...
+    </section>
 
-        <hr>
-        <header class="is-size-3 has-text-centered">Current Invitations:</header>
+    <div class="container">
 
-         {% for invitee in invitees %}
-            <div class="box">
-                <form class="is-inline-block" method="post" action="">
+        {% block content %}
```
Comment (Curtis-Barnhart) [inline our_site/planner/templates/planner/event_modify_invitees.html:27]: Is there a reason we have block content in here if there is no longer any inheritance?

Comment (Curtis-Barnhart) [reply our_site/planner/templates/planner/event_modify_invitees.html:27]: I guess this is a question about all the files which no longer use inheritance but still have block content?

File: our_site/planner/templates/planner/event_home_owned.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (Curtis-Barnhart) [reply our_site/planner/templates/planner/event_home_owned.html:1]: Or what would be even better is if we could put a conditional in the html so that it displays one thing slightly differently depending on who the user is.

File: our_site/planner/views.py [hunk 3]
```diff
@@ -187,6 +187,15 @@ def edit_event(request, event_id: int) -> HttpResponse:
                 event.description = posted_details.cleaned_data["event_description"]
                 event.location = posted_details.cleaned_data["event_location"]
                 event.save()
+                return HttpResponseRedirect("/planner/event_owned/{:d}/".format(event.id))
+
+    if request.method == "POST":
```
Comment (Curtis-Barnhart) [inline our_site/planner/views.py:192]: I think something might be up here  because we already have a place where the request method is checked and the form is messed around with already before we get here. This code should probably be inside the switch statement above.

File: our_site/planner/views.py [hunk 4]
```diff
@@ -204,6 +213,36 @@ def edit_event(request, event_id: int) -> HttpResponse:
     )
 
 
+def event_home_owned(request, event_id):
+    if request.user.is_anonymous:
```
Comment (Curtis-Barnhart) [inline our_site/planner/views.py:217]: There is a utility function event_specific_credentials created so that we don't have to copy and paste this code for checking to see if a user owns an event. It might need to be modified a little to get the Http404 error because at the moment it only does a redirect if they don't own it or aren't logged in, but that way we wouldn't have to write nearly as much code each time you want a function that validates the user in such a way.

Comment (Curtis-Barnhart) [reply our_site/planner/views.py:217]: I guess this isn't a change that needs to be made, just a note so that you don't have to do as much work next time.

**Review by Curtis-Barnhart — CHANGES_REQUESTED** [review]
The only things I think need to be changed are maybe all the {% block content %} places in html files that no longer use inheritance? If those aren't necessary, I think we should remove them (the other changes I suggested could be useful, but aren't really necessary since the site works fine as is and maybe instead we could just make tickets for them if we think they are important). Otherwise, looks good to me.

File: our_site/planner/templates/planner/event_modify_invitees.html (hunk 1 above)
Comment (mnugent04) [reply our_site/planner/templates/planner/event_modify_invitees.html:27]: got rid of them!

**Review by Curtis-Barnhart — APPROVED** [review]

**Review by gyishak — APPROVED** [review]
